numerics/src/tools/NumericsSparseMatrix.h

```diff
@@ -160,6 +160,9 @@ extern "C"
    */
   CS_INT* NSM_diag_indices(NumericsMatrix* M);
 
+  CS_INT* NSM_diag_indices_trivial(NumericsMatrix* M);
+
```

Comment:
Can NSM_diag_indices_trivial be a static function inside NumericsSparseMatrix.c?

That way it not in the ``public API'' and doesn't get used?

Reply:
I agree. This is done in the commit below